### DISCUSSION TITLE
Wired caption appears if we don’t provide caption to image and  shows different Preview image

### DISCUSSION THREAD

**Discussion by adidesai18** [kirimwa-id/forum#156] [Tanya Jawab] [answered]
1) While send message of type image ,in that if we not provide caption then it automatically sends a random date format message to target persons and the image which we send gets different preview .
2) And one more problem is that images are not able to sent in groups.

![IMG_98B60D580ADD-1](https://user-images.githubusercontent.com/94288784/209918079-b43914d3-60ee-4443-857a-bd60d75e2ef5.jpeg)
while downloading .

**Comment by rioastamal** [comment, marked as answer]
@adidesai9009 thanks for the report. 

1) I put this issue on the roadmap #157 and should be included in the next release. As for preview, the green thumbnail is intended. I have no plan to change it, since it require more processing resource to generate thumbnail.
2) I just tested and found no issue while sending image to a group. Please double check your request.

**Reply by rioastamal** [reply]
This bug is fixed on v1.0-RC20. See #20
